packages/core/admin/admin/src/components/GuidedTour/layout.js

```diff
@@ -0,0 +1,153 @@
+const layout = {
+  contentTypeBuilder: {
+    home: {
+      title: {
+        id: 'app.components.GuidedTour.home.CTB.title',
+        defaultMessage: '🧠 Build the content structure',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.home.CTB.cta.title',
+          defaultMessage: 'Go to the Content-type Builder',
+        },
+        type: 'REDIRECT',
+        target: '/plugins/content-type-builder',
+      },
+    },
+    create: {
+      title: {
+        id: 'app.components.GuidedTour.CTB.create.title',
+        defaultMessage: '🧠 Create a first Collection-type',
+      },
+      content: {
+        id: 'app.components.GuidedTour.CTB.create.content',
+        defaultMessage:
+          '<p>Collection-types help you manage several entries, Single-types are suitable to manage only one entry.</p> <p>Ex: For a website, articles would be a Collection type and homepage would be a Single type.</p>',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.CTB.create.cta.title',
+          defaultMessage: 'Build a Collection-type',
+        },
+        type: 'CLOSE',
+      },
+    },
+    success: {
+      title: {
+        id: 'app.components.GuidedTour.CTB.success.title',
+        defaultMessage: 'Step 1: Completed ✅',
+      },
+      content: {
+        id: 'app.components.GuidedTour.CTB.success.content',
+        defaultMessage: '<p>Good going!</p><b>⚡️ What would you like to share with the world?</b>',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.CTB.success.cta.title',
+          defaultMessage: 'Create sample content',
```

Comment:
```suggestion
          defaultMessage: 'Create content',
```

I'd avoid sample here, since the content might not be sample, but real content already.

Reply:
product copy - @iparidhi @maevalienard

Reply:
+1